docs/UserGuide.md

```diff
@@ -475,6 +499,10 @@ _Details coming soon ..._
 **Q**: Must I install java 17 or above to use InternBuddy? <br>
 **A**: Yes, InternBuddy uses libraries implemented in java 17 or above, it will not work without it.
 
+**Q**: Do I have to follow the order of parameters specified by the commands? <br>
+**A**: If the parameter has a prefix (eg: `n/NAME`), you can specify them in any order you like! However, parameters with no prefix (eg: `INDEX`)
+must appear as the first input after the command.
```

Comment:
Phrasing suggestion
`must appear as the first parameter after the command word.`